src/renderer/components/top-nav/top-nav.vue

```diff
@@ -61,6 +61,7 @@
           :is-search="true"
           :select-on-focus="true"
           :data-list="searchSuggestionsDataList"
+          spellcheck="false"
```

Comment:
You'll need to make this a dynamic assignment after making it expect a Boolean.
```suggestion
          :spellcheck="false"
```